### ISSUE TITLE
pipeline breaking at HTTPListenerRuleWithDomain after upgrading to copilot v1.27

### ISSUE THREAD

**Issue by nandubatchu** [aws/copilot-cli#4913]
Cloudformation Event Error for Logical-ID `HTTPListenerRuleWithDomain` : 
```
Resource handler returned message: "Resource of type 'AWS::ElasticLoadBalancingV2::ListenerRule' with identifier 'One or more priorities not found (Service: ElasticLoadBalancingV2, Status Code: 400, Request ID: <REDACTED>, Extended Request ID: null)' already exists." (RequestToken: <REDACTED>, HandlerErrorCode: AlreadyExists)

```

**Comment by nandubatchu** [comment]
when I bypass the pipeline with local copilot-cli command `copilot svc deploy` - it deployed successfully! 🤷 - pipeline is working fine post that!

**Comment by Lou1415926** [comment]
Hey @nandubatchu -  glad that your pipeline is green now!


https://github.com/aws/copilot-cli/issues/4532 comes to mind, in which I posted a solution in https://github.com/aws/copilot-cli/issues/4532#issuecomment-1444119556. 

TLDR - They also saw the same error message after they upgraded. If you have manually modified the listeners before, then it's likely that yours is the similar issue as theirs.

Let me know if the issue persist or resurface. We can dive deeper into it!
